core/src/main/java/org/geysermc/geyser/level/JukeboxSong.java

```diff
@@ -33,7 +33,13 @@ public record JukeboxSong(String soundEvent, String description) {
 
     public static JukeboxSong read(RegistryEntry entry) {
         NbtMap data = entry.getData();
-        String soundEvent = data.getString("sound_event");
+        Object soundEventObject = data.get("sound_event");
+        String soundEvent = "";
```

Comment:
Would rather set this as `String soundEvent;` and then set it as blank with a debug log if `soundEventObject` did not match either expected type.

Reply:
Added a debug log statement